brewchat/src/main/java/com/example/brewchat/ChatService.java

```diff
@@ -115,6 +122,29 @@ public void logout() {
         }
     }
 
+    public void addChatGroup(String name, ArrayList<Integer> userIds){
+        QBDialog dialog = new QBDialog();
+        dialog.setName(name);
+        dialog.setType(QBDialogType.GROUP);
+        dialog.setOccupantsIds(userIds);
+
+        QBGroupChatManager groupChatManager = QBChatService.getInstance().getGroupChatManager();
+        groupChatManager.createDialog(dialog, new QBEntityCallbackImpl<QBDialog>() {
+            @Override
+            public void onSuccess(QBDialog dialog, Bundle args) {
+                GroupChatCreatedEvent groupChatCreatedEvent = new GroupChatCreatedEvent(dialog.getName(), dialog.getOccupants());
+                EventBus.getDefault().post(groupChatCreatedEvent);
+                Log.d(TAG, "Created new Group Chat");
+            }
+
+            @Override
+            public void onError(List<String> errors) {
+                //Toast saying error in creating group chat
```

Comment:
Mark this comments with `TODO` or `FIXME` so we can search for them latter.

Please do not use a toast in this class. :)